pkgs/development/libraries/openssl/default.nix

```diff
@@ -8,7 +8,7 @@ let
   opensslCrossSystem = stdenv.cross.openssl.system or
     (throw "openssl needs its platform name cross building");
 
-  common = { version, sha256 }: stdenv.mkDerivation rec {
+  common = args@{ version, sha256, patches ? [] }: stdenv.mkDerivation rec {
```

Comment:
Why the `args@`? You can references the patches via `patches` instead of `args.patches`.